src/client/css/style.css

```diff
@@ -1703,7 +1703,7 @@ section#contact_form .text-danger {
   font-weight: 700;
   width: 200px;
   height: 50px;
-  margin: 0 0 0 22px;
+  /*margin: 0 0 0 20px;*/
```

Comment:
Take a look at how I make buttons center in the events section of the website. Maybe you find something there.